app/utility/base_planning_svc.py

```diff
@@ -113,6 +118,14 @@ async def obfuscate_commands(self, agent, obfuscator, links):
 
     """ PRIVATE """
 
+    @staticmethod
```

Comment:
personal preference is to have 'x' be 'link' or something descriptive.

Reply:
Yeah, that makes sense.